### PR TITLE
Bug 1790998: switch version metric to component-base

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,50 +3,34 @@ module github.com/openshift/oauth-server
 go 1.12
```
Comment (stlaz) [inline go.mod]: I think we want to switch this to `go 1.13`

**Comment by tnozicka** [comment]
/hold
the tests are failing because it is being built with go1.12 instead of 1.13
```--> RUN make build --warn-undefined-variables
build flag -mod=vendor only valid when using modules

**Pull request merged**